0010-B-object-storage-building-block.md

```diff
@@ -0,0 +1,264 @@
+# Object storage building block
+
+* Author(s): @ItalyPaleAle
+* State: Draft
+* Updated: 2022-12-14
+* Original proposals / discussions: dapr/dapr#4808, dapr/dapr#4934
+
+## Overview
```

Comment:
What is really missing from this proposal is the why (why users would care about this), the actual problems it helps solve and a mention of the alternatives. All these sections are indicated in the proposed template and need to be added here for clarity.

Reply:
Added

Reply:
Thanks, can you mention the alternatives? I have additional comments that would best go under that section.

Reply:
Besides Dapr state stores (which I sort-of mentioned), what other alternatives do you think are worth including? Are non-Dapr solutions (like libraries that abstract various object storage services) one thing I should include?

Reply:
Exactly, what would a user do without Dapr is important - it was included in other proposals and it serves as a really great way to know if the value add to the user with Dapr is worth both the maintenance effort for the project and also for the users who need to run it. (Dapr has overhead and we need to make sure APIs/features we add have a considerable value add).